scripts/close-without-repro.ts

```diff
@@ -0,0 +1,66 @@
+import { execSync } from "node:child_process";
+import { parseArgs } from "node:util";
+
+const sleep = (ms: number) =>
+  new Promise<void>((resolve) => setTimeout(resolve, ms));
+
+const { values: args } = parseArgs({
+  options: {
+    dryRun: {
+      type: "boolean",
+      default: false,
+    },
+  },
+  strict: true,
+});
+
+run();
+
+async function run() {
+  let issuesCmd = `gh issue list --search "is:issue state:open label:bug" --limit 250 --json number,body`;
+  console.log(`Executing command: ${issuesCmd}`);
+  let result = execSync(issuesCmd).toString();
+  let issues = JSON.parse(result) as { number: number; body: string }[];
+  let noReproIssues = issues.filter(({ body }) => {
+    return (
+      !/https?:\/\/stackblitz\.com\//.test(body) &&
+      !/https?:\/\/codesandbox\.io\//.test(body) &&
+      // Look for github links excluding user-uploaded screenshots
+      !/https?:\/\/github\.com\/(?!user-attachments)/.test(body)
+    );
+  });
```

Comment:
Decently reliable check for "having a reproduction" as those issues which contain a stackblitz/codesandbox/github repo link in the issue body.